cantor-auth/src/main/java/com/salesforce/cantor/grpc/auth/Role.java

```diff
@@ -0,0 +1,77 @@
+package com.salesforce.cantor.grpc.auth;
+
+import java.util.*;
+
+/**
+ * Cantor Authentication and Authorization is based on two main systems: {@link User} and {@link Role}
+ * <br><br>
+ * <h3>User:</h3>
+ * <p>A User can have any number of roles assigned to them. Created to be the backend's access management tool, this is
+ * meant to reduce the scope of a user's activities or track usage.</p>
+ * <br>
+ * <h3>Role:</h3>
+ * <p>Role defines the read and write access of individual namespaces. Both lists will accept regex patterns to match
+ * against namespaces.</p>
+ */
+public class Role {
+    public static final String ROLES_NAMESPACE = ".roles";
+    public static final List<String> WRITE_METHODS = Arrays.asList("store", "delete", "expire", "add", "create", "drop");
```

Comment:
@AlexanderKouthoofd instead of having only read/write access, allow control over individual methods.